deployments/helm/nvidia-device-plugin/templates/daemonset-gfd.yml

```diff
@@ -199,6 +201,10 @@ spec:
         - name: host-sys
           hostPath:
             path: "/sys"
+        - name: nvidia-imex-dir
+          type: DirectoryOrCreate
+          hostPath:
+            path: {{ include "nvidia-device-plugin.filepathJoin" (list .Values.nvidiaDriverRoot "etc/nvidia-imex") }}
```

Comment:
This still doesn't include `/` as the first element in the list to ensure that it is an absolute path.

Reply:
added